rollbar.go

```diff
@@ -169,23 +173,28 @@ func buildBody(level, title string) map[string]interface{} {
 	timestamp := time.Now().Unix()
 	hostname, _ := os.Hostname()
 
+	data = map[string]interface{}{
```

Comment:
good catch, I've submitted pull/16 with exactly that fix. Sorry. I ran `go build` before deciding to add the if logic to keep `code_version` logic in line with current use and didn't run it again after :-/